global.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
     "sdk":  {
-                "version":  "6.0.401",
+                "version":  "6.0.402",

```

Comment:
Likewise, I don't think this should be changed either. This implies the lowest acceptable SDK version, as the next line directs to roll forward to the next available feature version, if available.